diesel/src/expression/mod.rs

```diff
@@ -37,6 +37,7 @@ pub(crate) mod nullable;
 #[macro_use]
 pub(crate) mod operators;
 mod case_when;
+pub mod cast;
```

Comment:
I would like to keep the module private and just reexport the necessary items from the `expression` module.